src/index.html

```diff
@@ -10,12 +10,12 @@
 </head>
 <body>
   <app-root></app-root>
-  <!-- 
-  
+  <!--
+
   Ideally, replace this with an audio component. And give it a better name.
   Example uses video component, found here:
-  https://github.com/webrtc/samples/blob/gh-pages/src/content/devices/input-output/index.html 
+  https://github.com/webrtc/samples/blob/gh-pages/src/content/devices/input-output/index.html
 -->
-<video id="video-control-for-media-control-provider" playsinline autoplay style="visibility: collapse;"></video>
+<video id="video-control-for-media-control-provider" playsinline autoplay style="display: none;"></video>
```

Comment:
This fixes the unwanted scrolling at the end of the layout.

Reply:
Even better!